docs/realtime.rst

```diff
@@ -0,0 +1,60 @@
+.. |br| raw:: html
+
+   <br />
+
+.. _realtime:
+
+Real-time
+#########
+
+Fast RTPS can be configured to offer real-time features.
+These features will guarantee Fast RTPS responses within specified time constrains.
+To maintain this compromise Fast RTPS is able to have next behavior:
+
+- Not allocate memory after the initialization of Fast RTPS entities.
+- Several methods are blocked for a maximum period of time.
+
+This section explains how to configure Fast RTPS to achieve this behavior.
+For easier understanding it was divided in two subsections:
+
+- :ref:`allocations`: configuration to avoid memory allocation after initialization.
+- :ref:`non-blocking-calls`: usage of non-blocking methods for real-time behavior.
+
+.. _allocations:
+
+Allocations
+***********
+
+.. _non-blocking-calls:
+
+Non-blocking calls
+******************
+
+**Note:** This feature is not fully supported on OSX.
+It doesn't support necessary POSIX Real-time features.
+The feature is limited by the implementation of `std::timed_mutex` and `std::condition_variable_any`.
```

Comment:
So that this can be merged as part of release 1.9.0, should we also mention that at this stage `AsyncThread` and `UDP` transport are not supported yet?